packages/storybook-vue/stories/components/segmented-button/ScaleSegmentedButton.vue

```diff
@@ -26,6 +29,9 @@ export default {
     helperText: { type: String },
     fullWidth: { type: Boolean, default: false },
     styles: { type: String },
+    selectedIndex: { type: Number },
+    ariaLabelTranslation: { type: `segment button with $slottedSegments` },
```

Comment:
also here the prop description could help 
https://github.com/telekom/scale/blob/8a04a5e8c0bb3b72a2fbb90822600f6b8feefc06/packages/components/src/components/segmented-button/readme.md?plain=1#L12